scripts/dev-cli.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# turn on bash's job control
+set -m
+
+# start daemon
+mesg-daemon start &
+daemon=$!
+
+# wait 5 sec for the daemon to start rpc server
+sleep 5 &
+wait $!
```

Comment:
Isn't this sufficient?
```suggestion
sleep 5
```

Reply:
see answer https://github.com/mesg-foundation/engine/pull/1806#discussion_r421927943